src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -253,7 +254,7 @@ private void ValidateTypeForAttributeParameters(ImmutableArray<ParameterSymbol>
             }
         }
 
-        protected bool IsAttributeConditionallyOmitted(NamedTypeSymbol attributeType, SyntaxTree syntaxTree, ref HashSet<DiagnosticInfo>? useSiteDiagnostics)
+        protected bool IsAttributeConditionallyOmitted(NamedTypeSymbol attributeType, SyntaxTree? syntaxTree, ref HashSet<DiagnosticInfo>? useSiteDiagnostics)
```

Comment:
I'm not sure how this change is correct or how we're not warning on the dereference below.

Reply:
It isn't a dereference.  It is an extension method that accepts a null receiver.

---
In reply to: [387951748](https://github.com/dotnet/roslyn/pull/42068#discussion_r387951748) [](ancestors = 387951748)